test/pack.test.ts

```diff
@@ -76,118 +82,103 @@ describe("Pack Contract", async () => {
     });
 
     const packTwo = await packContract.create({
-      assetContract: bundleContract.getAddress(),
-      assets: [
+      erc1155Rewards: [
         {
+          contractAddress: bundleContract.getAddress(),
           tokenId: "0",
-          amount: BigNumber.from(50),
+          quantityPerReward: 1,
+          totalRewards: 50,
         },
         {
+          contractAddress: bundleContract.getAddress(),
           tokenId: "1",
-          amount: BigNumber.from(50),
+          quantityPerReward: 1,
+          totalRewards: 50,
         },
         {
+          contractAddress: bundleContract.getAddress(),
           tokenId: "2",
-          amount: BigNumber.from(50),
+          quantityPerReward: 1,
+          totalRewards: 50,
         },
       ],
       metadata: {
```

Comment:
should we call this "packMetadata" to be clear its for the pack?

Reply:
yep good idea, maybe should to something similar for MW